### PR TITLE
M3-5870: Marketplace App Details Drawer UI Refinement

### PR DESCRIPTION
**Pull request by bnussman**
## Description 📝

- Fixes lack of drawer animation on first open
- New UI for Marketplace App Details Drawer
- Schema changes to FakeSpec 
  - Adds `colors`
  - Removes `related_info` in favor of `website`
- Adds white icon svgs
  - I considered a css filter over our existing colored icons, but it created some undesirable distortion to some icons
  - I also considered linking the white svgs from linode.com directly, but I figured we'd prefer to not have the dependency on an external site

## Preview 🔎 

Screen Shot 2022-08-01 at 4 23 17 PM

## How to test 🧪

- Test the details drawer for Marketplace apps (`http://localhost:3000/linodes/create?type=One-Click`)

### CODE REVIEW

File: packages/manager/src/features/OneClickApps/AppDetailDrawer.tsx [hunk 1]
```diff
@@ -61,78 +90,119 @@ export const AppDetailDrawer: React.FunctionComponent<Props> = (props) => {
...
+            <Box className={classes.container}>
+              <Typography variant="h3">{app.summary}</Typography>
+              <Typography
+                variant="body1"
+                className={classes.description}
+                dangerouslySetInnerHTML={{
+                  __html: sanitizeHTML(app.description),
+                }}
+              />
+              {app.website && (
```
Comment (DevDW) [inline packages/manager/src/features/OneClickApps/AppDetailDrawer.tsx:151]: We can go ahead and give these the ternary treatment (`abc ? def : null`)

**Review by DevDW — COMMENTED** [review]
Github is being weird at the moment and not allowing me to see the diff for `FakeSpec.ts`, but this is looking good from what I can see so far

File: packages/manager/src/features/OneClickApps/AppDetailDrawer.tsx [hunk 2]
```diff
@@ -61,78 +90,119 @@ export const AppDetailDrawer: React.FunctionComponent<Props> = (props) => {
...
     const cleanedAppName = app.name.replace('&reg;', '');
     return cleanedStackScriptLabel === cleanedAppName;
-  }); // This is horrible
-  if (!app) {
-    return null;
-  }
+  });
+
+  const gradient = {
+    backgroundImage: `url(https://www.linode.com/wp-content/themes/linode-website-theme/images/Triangles-Background--Dark--Desktop.png),linear-gradient(to right, #${app?.colors.start}, #${app?.colors.end})`,
```
Comment (jdamore-linode) [inline packages/manager/src/features/OneClickApps/AppDetailDrawer.tsx:96]: Do we want our own copy of the background image?

**Review by jdamore-linode — APPROVED** [review]
This is looking great so far! Nice work Banks!

File: packages/manager/src/features/OneClickApps/AppDetailDrawer.tsx (hunk 2 above)
Comment (bnussman) [reply packages/manager/src/features/OneClickApps/AppDetailDrawer.tsx:96]: Yes, good idea.

**Review by DevDW — APPROVED** [review]
🚀

**Pull request merged**